### DISCUSSION TITLE
Font size?

### DISCUSSION THREAD

**Discussion by Solenhal** [microsoft/terminal#18241] [Q&A]
When using Powershell 7.4.6 how do I change the font size? Readability is important.

**Comment by zadjii-msft** [comment]
* you can `ctrl+mouse wheel` to change the font size
* you can use the [`adjustFontSize`](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#adjust-font-size) action, bound to `ctrl+plus` and `ctrl+minus` by default
* You can adjust the settings for the profile in the settings, under appearance 
  ![image](https://github.com/user-attachments/assets/f9f21c2e-d544-4cdd-af3a-8cc11de5c845)
